src/api/client.ts

```diff
@@ -438,8 +424,6 @@ class ApiClient {
     );
   }
 
-  // NOTE: This won't actually get all incidents anymore because of the pagination
```

Comment:
seems like a reasonable comment tbh, name is misleading and the comment corrects it (ASSUMING that this comment is even true, I have no idea. if its not true, then ofc remove it). Best would be to make the name less misleading, but since the next comment is "// TODO: Remove all use of this method.", its probably not worth putting energy into renaming if its gonna be removed soon.

Reply:
This comment is outdated